gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -464,28 +464,29 @@ MM_MemorySubSpaceSemiSpace::tearDown(MM_EnvironmentBase *env)
 void
 MM_MemorySubSpaceSemiSpace::flip(MM_EnvironmentBase *env, Flip_step step)
 {
-#if defined(OMR_GC_CONCURRENT_SCAVENGER)
-	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
-	bool debug = _extensions->debugTiltedScavenge;
-#endif
 	switch (step) {
 	case set_evacuate:
+		Trc_MM_MSSSS_flip_status(env->getLanguageVMThread(), "set_evacuate");
```

Comment:
perhaps you can call it _step, but _status is ok too

Reply:
Done :)